src/Command/InstallModxCommand.php

```diff
@@ -35,6 +35,12 @@ protected function configure()
                 'The version of MODX to install, in the format 2.3.2-pl. Leave empty or specify "latest" to install the last stable release.',
                 'latest'
             )
+            ->addArgument(
+                'configFile',
```

Comment:
```suggestion
                'config',
```
Change to config, instead configFile